NOTICE.txt

```diff
@@ -551,46 +658,6 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 ---
 
-## wix
```

Comment:
We still use WiX for creating the MSI build. Can't remove this.

Reply:
I added it back. It's not listed here though: https://github.com/mattermost/desktop/blob/master/package.json#L173.